docs/configuration.rst

```diff
@@ -43,6 +43,11 @@ A few relevant settings for your `Pyramid .ini file <http://docs.pylonsproject.o
         # Default: True
         pyramid_swagger.enable_path_validation = True
 
+        # Use Python classes instead of dicts to represent models in incoming
+        # requests.
+        # Default: Whatever the default is in bravado_core, currently True
```

Comment:
Can we explicitly default this inside pyramid_swagger code (e.g. if it's unset, set it false and pass that explicitly into bravado_core)? I wouldn't want this default to secretly change on users without an obvious code change here.